bot.go

```diff
@@ -338,6 +339,20 @@ func updateMessage(session *discordgo.Session, message *discordgo.MessageUpdate)
 	handleMessage(session, &discordgo.MessageCreate{Message: message.Message}, reply)
 }
 
+func deleteMessage(session *discordgo.Session, message *discordgo.MessageDelete) {
+	log.S(log.Debug, "message delete", log.Any("message", message), log.Any("before", message.BeforeDelete))
+	reply, found := msgSet.Get(message.ID)
+	if !found {
+		log.S(log.Debug, "message not handled before", log.Any("id", message.ID))
+		return
+	}
+	log.S(log.Info, "message delete detected",
+		log.Any("id", message.ID),
+		log.Any("reply", reply),
+		log.Any("before", message.BeforeDelete))
+	handleMessage(session, &discordgo.MessageCreate{Message: message.Message}, reply)
```

Comment:
So a message deletion? Call a messageCreate ?

I'm asking because it looks weird

Reply:
create, update, and now delete share the same logic, which is that create doesn’t have a reply already and update/delete both (can for update, does for delete) delete replies

it’s a bit overkill but also avoids duplication

Reply:
I guess I could probably pass only the inner message to handle message

Reply:
#63